Apt.py

```diff
@@ -0,0 +1,66 @@
+"""Apt extension
+
+Note that it is up to you to ensure that the package lists are up to date"""
+
+import os
+import subprocess
+from shutil import which
+from albertv0 import *
+import re
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Apt"
+__version__ = "1.0"
+__trigger__ = "apt"
+__author__ = "Harindu Dilshan"
+__dependencies__ = ['apt','gksu','pkexec']
+
+if which("apt") is None:
+    raise Exception("'Apt' is not in $PATH.")
+
+gksu = None
+for item in __dependencies__:
+    if which(item):
+        gksu = item
+
+UpdateActions = [TermAction("Update database in Terminal", ["sudo", "apt", "update"])]
+
+if gksu is not None:
+    UpdateActions.insert(0,FuncAction("Update database",lambda :subprocess.Popen([gksu,"apt","update"])))
+
+iconPath = iconLookup("system-software-install")
+
+def handleQuery(query):
+    results = []
+    if query.isTriggered:
+        if len(query.string) > 2:
+            # pattern = re.compile(query.string, re.IGNORECASE)
+            shell_cmd = ["apt-cache" ,"search"]
```

Comment:
apt-cache is not in the deps list. I am not good with debian. If it is always installed thats fine. if not users should know it.